tools_hive/conf/vep_plugins_hive_config.txt

```diff
@@ -41,7 +41,8 @@
 
   CADD => {
     "params" => [
-      "[[ENSEMBL_VEP_PLUGIN_DATA_DIR]]/CADD_GRCh38_1.5_whole_genome_SNVs.tsv.gz"
+      "[[ENSEMBL_VEP_PLUGIN_DATA_DIR]]/CADD_GRCh38_1.5_whole_genome_SNVs.tsv.gz",
+      "[[ENSEMBL_VEP_PLUGIN_DATA_DIR]]/CADD_GRCh38_1.5_Indels.tsv.gz",
```

Comment:
Hi @ima23, I just wanted to make sure that the trailing comma here is fine. Has this been tested already?

Reply:
Hi @msmanoj, thanks for checking, the comma should be fine, I have removed it in the next commit. Tested it on rest.